src/modules/transactions/hooks/list/useGetPendingTransactionsList.ts

```diff
@@ -0,0 +1,32 @@
+import { ITransactionsGroupedByMonth } from '../../services';
+import { IPendingTransactionsRecord } from './useTransactionList';
+
+const usePendingTransactionsList = (
+  homeTransactions: ITransactionsGroupedByMonth[],
+  transactionsPageList: ITransactionsGroupedByMonth[],
+): IPendingTransactionsRecord => {
+  const mergedTransactionsList =
+    homeTransactions && transactionsPageList
+      ? homeTransactions.concat(transactionsPageList)
+      : [];
+
+  const result = {};
+  mergedTransactionsList?.forEach((item) => {
```

Comment:
replace this after change api transaction list return
api -> return an array, with tx -> [{tx_1}, {tx_2}, {tx_3}]
hook -> using a reduce, group this transactions by montly (keep on easy import local)

Reply:
i fixed this logic in the branch specific for grouped transactions logic.